src/pybind/mgr/dashboard_v2/controllers/dashboard.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import cherrypy
+
+from ..tools import ApiController, AuthRequired, BaseController, RemoteViewCache
+
+
+@ApiController('dashboard')
+@AuthRequired()
+class Dashboard(BaseController):
+    def __init__(self):
+        super(Dashboard, self).__init__()
+        self.rbd_pool_ls = RbdPoolLs(self.mgr)
+
+    @cherrypy.expose
+    @cherrypy.tools.json_out()
+    def toplevel(self):
+        _, data = self.rbd_pool_ls.get()
+        if data is None:
+            self.mgr.log.warning("Failed to get RBD pool list")
+            data = []
+        data.sort()
+        return {
+            'rbd_pools': data
+        }
```

Comment:
`return sorted(data)` ? No need for wrapping that in a dict, right?

Reply:
This is a dict because https://github.com/openattic/ceph/pull/31 will add more field to this dict.